install/kubernetes/helm/istio-init/files/crd-14.yaml

```diff
@@ -1,26 +1,141 @@
-kind: CustomResourceDefinition
 apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
nit: for clean history can you only modify what is needed

Reply:
This is a generated file. So I believe the generated yaml respects the order of fields in k8s api.